server/src/main/java/com/exacaster/lighter/storage/ApplicationStorage.java

```diff
@@ -18,4 +20,8 @@ public interface ApplicationStorage {
 
     List<Application> findApplicationsByStates(ApplicationType type, List<ApplicationState> states, SortOrder order,
             Integer from, Integer size);
+
+    List<PermanentSession> findAllPermanentSessions();
```

Comment:
I think it is nicer to not have `ApplicationType` specific methods here. Maybe we could add `findAllApplications(ApplicationType type)` instead?

Reply:
done